File: aoc2017/day2/part1.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
	"sort"
	"strconv"
	"strings"
)

func main() {
	file, err := os.Open("./input.txt")
	if err != nil {
		log.Fatal(err)
	}

	defer file.Close()

	scanner := bufio.NewScanner(file)
	count := 0
	diff := 0
	for scanner.Scan() {
		var valSl []int
		vals := strings.Split(scanner.Text(), "\t")
		for _, val := range vals {
			v, e := strconv.Atoi(val)
			if e != nil {
				log.Fatal(e)
			}
			valSl = append(valSl, v)
		}
		sort.Ints(valSl)
		diff = valSl[len(valSl)-1] - valSl[0]
		count += diff
	}
	fmt.Println(count)

}

File: aoc2021/day1/part1.py

def main():
    with open('input.txt', 'r') as f:
        p = f.read().split('\n')
    p = map(lambda x: int(x), p)
    p = list(p)
    counter = 0

    for i in range(1, len(p)):
        if p[i] > p[i-1]:
            counter += 1
    print(counter)

if __name__ == '__main__':
    main()
